src/util.tcc

```diff
@@ -25,11 +26,11 @@ template<typename T> T swap_bit_endianness(T v)
     return v;
 }
 
-// Takes a binary string and swaps the byte endianness
-template<typename FieldT>
-std::vector<FieldT> swap_byte_endianness(std::vector<FieldT> value)
+// Takes a containers with a size method and reverse the elements' order
+// The elements should represent bits
```

Comment:
Shouldn't it be "bytes" instead of "bits"?

Reply:
We swap arrays of bits (if you look at the code.

```
    if (len % 8 != 0) {
        throw std::length_error("Invalid bit length for the given boolean "
                                "vector (should be multiple of 8)");
    }
```

Reply:
Indeed